tests/e2e-pw/specs/merchant/merchant-admin-analytics.spec.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import * as shopper from '../../utils/shopper';
+import { config } from '../../config/default';
+import { getMerchant, getShopper, useMerchant } from '../../utils/helpers';
+import {
+	activateMulticurrency,
+	ensureOrderIsProcessed,
+	isMulticurrencyEnabled,
+	setOption,
+	tableDataHasLoaded,
+} from '../../utils/merchant';
+import { goToOrderAnalytics } from '../../utils/merchant-navigation';
+
+test.describe( 'Admin order analytics', () => {
+	let orderId: string;
+
+	// Use the merchant user for this test suite.
+	useMerchant();
+
+	test.beforeAll( async ( { browser } ) => {
+		const { shopperPage } = await getShopper( browser );
+		const { merchantPage } = await getMerchant( browser );
+
+		if ( false === ( await isMulticurrencyEnabled( merchantPage ) ) ) {
+			await activateMulticurrency( merchantPage );
+		}
+
+		// Place an order to ensure the analytics data is correct.
+		await shopperPage.goto( '/cart/' );
+		await shopper.addCartProduct( shopperPage );
+
+		await shopper.setupCheckout(
+			shopperPage,
+			config.addresses.customer.billing
+		);
+		await shopper.fillCardDetails( shopperPage, config.cards.basic );
+		await shopper.placeOrder( shopperPage );
+
+		// Get the order ID
+		const orderIdField = shopperPage.locator(
+			'.woocommerce-order-overview__order.order > strong'
+		);
+
+		orderId = await orderIdField.innerText();
```

Comment:
We can use `placeOrderWithCurrency` from the shopper utils to replace these lines with a single one 🙂

Reply:
Didn't know about that one :) Thanks for the suggestion. I'll replace that with the next commit.

Reply:
Replaced in [e184323](https://github.com/Automattic/woocommerce-payments/pull/10127/commits/e184323045505ff3d72304fe6374c8fe55f6ea85)